charts/staffeln/templates/openrc.tpl

```diff
@@ -0,0 +1,30 @@
+{{/*
+Defines openrc for Staffeln.
+*/}}
+
+{{- define "staffeln.templates.openrc" -}}
+export OS_IDENTITY_API_VERSION=3
+export OS_AUTH_TYPE=password
+export OS_INTERFACE=internal
+{{- if .Values.conf.clouds.clouds.envvars.auth.auth_url }}
+export OS_AUTH_URL="{{ .Values.conf.clouds.clouds.envvars.auth.auth_url }}"
+{{- end }}
+{{- if .Values.conf.clouds.clouds.envvars.auth.project_name }}
+export OS_PROJECT_NAME={{ .Values.conf.clouds.clouds.envvars.auth.project_name }}
+{{- end }}
+{{- if .Values.conf.clouds.clouds.envvars.auth.project_domain_name }}
+export OS_PROJECT_DOMAIN_NAME={{ .Values.conf.clouds.clouds.envvars.auth.project_domain_name }}
+{{- end }}
+{{- if .Values.conf.clouds.clouds.envvars.region_name }}
+export OS_REGION_NAME="{{ .Values.conf.clouds.clouds.envvars.region_name }}"
+{{- end }}
+{{- if .Values.conf.clouds.clouds.envvars.auth.user_domain_name }}
+export OS_USER_DOMAIN_NAME={{ .Values.conf.clouds.clouds.envvars.auth.user_domain_name }}
+{{- end }}
+{{- if .Values.conf.clouds.clouds.envvars.auth.password }}
+export OS_PASSWORD="{{ .Values.conf.clouds.clouds.envvars.auth.password }}"
+{{- end }}
+{{- if .Values.conf.clouds.clouds.envvars.auth.username }}
+export OS_USERNAME="{{ .Values.conf.clouds.clouds.envvars.auth.username }}"
```

Comment:
Could you check if we can use `helm-toolkit.snippets.keystone_secret_openrc` to generate openrc using htk?
Also instead of setting cloud config in `values.conf.clouds.clouds.envvars` , we fetch from `values.endpoints` and it is more helm-ly way i think.
You can refer to `secret-keystone.yaml` file of other charts.

Reply:
it does not goes in directly, but let me do some string replace and put it in followup pr

Reply:
I remember why I didn't use keystone_secret_openrc at first place. Because staffeln might use different account to format openrc here, and we need be allow it reset